### ISSUE TITLE
Application crashes when uploading POIs to OSM

### ISSUE THREAD

**Issue by Zirochkabila** [osmandapp/OsmAnd#15043]
### Description
Application crashes when uploading POIs to OSM
[exception (8).log.zip](https://github.com/osmandapp/OsmAnd/files/9289715/exception.8.log.zip)

### Steps to reproduce
Menu > My places > OSM EDITS  > Select all > Upload all > Upload

### Actual result
App is crashed

https://user-images.githubusercontent.com/104760013/183631033-0f89355d-8fdd-4e9b-aef3-abf2de1fd59f.mp4

### Expected result
A message that the POI has been successfully downloaded

### Your Environment
OsmAnd Version: OsmAnd+ ver. 4.2.7
Android/iOS version: Android 11
Device model: Nokia 5.3

**Comment by gabisktroll** [comment]
Exactly the same problem, with the same steps.

Log file
[exception.log](https://github.com/osmandapp/OsmAnd/files/9300190/exception.log)

Device : sweetBrand, note 10 pro : RedmiModel : M2101K6GProduct : sweet_globalBuild : SKQ1.210908.001 test-keysVersion : 12App Version : OsmAnd+Apk Version : 4.2.7 4207
 But also with my Samsung A12

**Comment by kadlecf** [comment]
Using the module, I am mostly able to upload single POIs; exceptionally, I notice a crash there too. Namely, I recorded one crash with a POI upload from an area whose detailed map I had deleted (only the world map was available), and I was uploading the POI from the map view via the dialog in the lower part of the screen. But I am not sure this is 100% reproducible.

OsmAnd Version: OsmAnd+ ver. 4.2.7
Android/iOS version: Android 11
Device model: Nokia 5.3
